app/models/proposal.rb

```diff
@@ -190,7 +219,8 @@ def version
   def restart
     # Note that none of the state machine's history is stored
     self.api_tokens.update_all(expires_at: Time.now)
-    self.initialize_approvals()
+    self.approvals.update_all(status: 'pending')
+    self.kickstart_approvals()
```

Comment:
This just maintains the existing logic, which is convoluted. Previously, `initialize_approvals` meant resetting all approvals to the pending state and then triggering the first (or all, if parallel). We don't want to `reset_status` as that's already accounted for in the state transition.